for/wordpress.md

```diff
@@ -0,0 +1,193 @@
+
+```bash
+wpcept generate:wpunit wpunit "Acme\SomeClass"
+```
+
+The generated test case extends the `Codeception\TestCase\WPTestCase` class and it exposes all the mehtods defined by `Codeception\TestCase\Test` test case and the Core suite `WP_UnitTestCase`.  
+Additional test method generation possibilities are available to cover the primitive test cases offered in the Core testing suite: `wpajax`, `wprest`, `wpcanonical`, `wpxmlrpc`.  
+Any database interaction is wrapped in a transaction to guarantee isolation between tests.
+
+<div class="alert alert-warning">
+  <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+  Continue to <a href="http://codeception.com/docs/05-UnitTests">Unit Testing Guide &raquo;</a>.
+</div>
+
+## WordPress Functional Tests
+Functional tests are meant to test complex interactions between a WordPress theme or plugin components, requests handling or a filter triggered chain of events.  
+A WordPress unit test might test the behaviour of a post insertion handling class in isolation where a functional test might verify the behaviour of said class when part of a chain of classes handling a post insertion request.  
+Similarly to WordPress Unit Tests the `functional` suite should include the `WPLoader` module to have WordPress running in the same scope as the test methods and the configuration is the same.
+Since the difference is subtle in infrastructure terms the command to generate a functional test is the same used to generate a WordPress unit test:
+
+```bash
+wpcept generate:wpunit functional "PostInsertionHandling"
+```
+
+## Acceptance Tests
+
+To test a WordPress theme or plugin functionalities using its UI you should simulate the user interaction with a browser both in the site front-end and back-end.  
+You can use the `PHPBrowser`, `WebDriver` and `Db` modules defined by Codeception but the WPBrowser package extends those modules in the `WPBrowser`, `WPWebDriver` and `WPDb` modules to add WordPress specific methods to each.  
+The `WPBrowser` module does not support JavaScript in the same way the `PHPBrowser` module doesn't and the `WPWebDriver` module drives a real browser in the same way the `WebDriver` does.  
+The modules require some WordPress specific parameters to work:
+
+```yaml
+modules:
+    enabled:
+        WPBrowser:
+            url: 'http://wordpress.dev'
+            adminUsername: admin
+            adminPassword: password
+            adminUrl: /wp-admin
+        WPDb:
+            dsn: 'mysql:host=localhost;dbname=wordpress'
+            user: root
+            password: root
+            dump: tests/_data/wp.sql
+            populate: true
+            cleanup: true
+            url: 'http://wordpress.dev'
+            tablePrefix: wp_
+        WPWebDriver:
```

Comment:
modules WPBrowser and WPWebDriver must not be used together as they share same interface but different backend. I think turning them both on in Codeception 2.2 will throw a conflict exception.

I'd recommend to use `WPBrowser` in functional test suite and `WPWebDriver` in acceptance test suite. The difference would be that functional test functionality and acceptance would test UI. We can even call that suite `ui` or `e2e` for better understanding.